src/subscriptions.ts

```diff
@@ -42,36 +42,39 @@ export async function recoverCredsIfRequired(orderId: string): Promise<void> {
     const sdk = await loadRewardsSdk();
 
     currentMethod = "refresh_order";
+    log("calling refresh_order...");
     const order = await sdk.refresh_order(orderId);
 
     currentMethod = "credential_summary";
+    log("calling credential_summary...");
     const summary = await sdk.credential_summary(order.location);
     if (["paid", "canceled"].includes(order.status)) {
       if (!summary) {
         currentMethod = "fetch_order_credentials";
+        log("calling fetch_order_credentials...");
         await sdk.fetch_order_credentials(orderId);
       }
     } else {
       throw new Error("Order not paid.");
     }
   } catch (e) {
-    console.error(`brave-rewards-js-sdk: ${currentMethod} fails`, e);
+    error(`${currentMethod} fails`, e);
     throw e;
   }
 }
 
 export async function checkSubscribedUsingSDK(): Promise<boolean> {
   try {
     const sdk = await loadRewardsSdk();
-    console.log(`brave-rewards-js-sdk: calling credential_summary...`);
+    log(`calling credential_summary...`);
 
     const result = await sdk.credential_summary();
-    console.log("brave-rewards-js-sdk: credential_summary returns", result);
+    log("credential_summary returns", result);
     if (result) {
```

Comment:
credential_summary now returns even in cases where the credential has expired, you'll want to change this to `result && result.active`

Reply:
Oh yes, you did say that. Will update, thanks.

Reply:
updated in #72